content/influxdb/cloud/tools/thingworx.md

```diff
@@ -0,0 +1,90 @@
+{{% /note %}}
+3. [Create an All-Access token](/influxdb/v2.0/security/tokens/create-token/) in InfluxDB Cloud, and save the token string for step 4. To access this string in the UI, double-clicking the new token name, and copy the string at the top of the dialog.
+4. Create a DBRP mapping for your bucket by running the following command, replacing all parameters within the brackets (`${}`) below:
+
+    ```sh
+    curl --request POST "${influxdb-cloud-url}/api/v2/dbrps" \
+    --header 'Content-Type: application/json' \
+    --header "Authorization: Token ${token}" \
+    --data-raw "{
+        \"bucket_id\": \"${bucket_id}\",
+        \"organization_id\": \"${org_id}\",
+        \"database\": \"thingworx",
+        \"retention_policy\": \"autogen\",
+        \"default\": true
+    }"
+    ```
+
+   - `bucket_id` created in step 2
+   - `token` created in step 3
+   - `org-id` in your URL (for example, if your URL includes: `.../orgs/039ax0d07d962000`, the org ID is `039ax0d07d962000`)
+   - `database` same as bucket name created in step 2
+   - `retention_policy` is the retention period, typically "autogen"
+
+5. (Optional) We recommend [creating a new Read/Write token](/influxdb/v2.0/security/tokens/create-token/) with read/write access to the bucket that you set up in step 2. A read/write token is useful for non-admin users to access configuration settings in ThingWorx. Copy and save the new token string to [set up PTC ThingWorx](#set-up-ptc-thingworx). (To do this, double-click the token in the UI, and copy the string at the top of the dialog.)
+
+## Set up PTC ThingWorx
+
+1. If you haven't already, [start at step 4 in Using InfluxDB as the Persistence Provider](https://support.ptc.com/help/thingworx_hc/thingworx_8_hc/en/index.html#page/ThingWorx%2FHelp%2FComposer%2FDataStorage%2FPersistenceProviders%2Fusing_influxdb_as_the_persistence_provider.html). (Steps 1-3 are covered being InfluxDB Cloud is already configured.)
+2. For the persistence provider configuration settings, enter the following values:
+
+    - **Connection URL**: Your [InfluxDB Cloud region URL](/influxdb/cloud/reference/regions)
+    - **Database Schema**: Database name in the DBRP mapping set up in step 4 above.
+    - **Username**: Login email address for your InfluxDB Cloud account.
+    - **Password**: Token string--either All-Access created in step 3 or Read/Write created in step 5.
+
+## Use the InfluxDB 1.x compatibility API on InfluxDB Cloud
+
+Use [InfluxDB 1.x compatibility API](/influxdb/cloud/reference/api/influxdb-1x/) to access the InfluxDB v2 API on InfluxDB Cloud, and include the InfluxDB persistence provider configuration settings you set up for PTC ThingWorx in your API requests.
+
+**To write data**
+
+```sh
+curl -XPOST "https://westeurope-1.azure.cloud2.influxdata.com/write?db=${database}" \
```

Comment:
Being we say only use this method with GCP or AWS...is the azure link misleading (or maybe am missing something)?

Reply:
thank you @gunnaraasen - deleted